File: android/src/main/java/au/com/crazybean/mobilex/kurir/modules/tasks/TasksView.kt
package au.com.crazybean.mobilex.kurir.modules.tasks

import au.com.crazybean.mobilex.kurir.data.model.Task
import au.com.crazybean.mobilex.kurir.modules.base.View

interface TasksView : View {
    fun showSpinner()
    fun hideSpinner()
    fun showTasks(tasks: List<Task>)
    fun showEmpty()
    fun showCreation()
}

File: ios/Kurir/modules/tasks/TasksView.swift
//
//  FindView.swift
//  Kurir
//
//  Created by Loren on 19/9/19.
//  Copyright © 2019 Crazybean Studio. All rights reserved.
//

import Mobilex

protocol TasksView {
    func showSpinner()
    func hideSpinner()
    func showTasks(tasks: [Task])
    func showEmpty()
}
